### ISSUE TITLE
Geforce NOW running via WINE crashes on startup

### ISSUE THREAD

**Issue by simonvanderveldt** [lutris/lutris#3263]
I'm not entirely sure this is the correct repo, but I couldn't find a repo with the script anywhere, so I figured it must be this one.

Geforce NOW as installed by the install script from here https://lutris.net/games/geforce-now/ (or here https://lutris.net/games/install/18981/view, I don't know if this is a version pinned view of the script) no longer works, it crashes on startup whilst showing a black window.

**Steps to reproduce**
- Install Geforce NOW using the script from https://lutris.net/games/geforce-now/
- Start Geforce NOW and see it crash

Complete log
```
Running /home/simon/.local/share/lutris/runners/wine/lutris-gfn-6-x86_64/bin/wine /home/simon/Games/geforce-now/drive_c/users/simon/Local Settings/Application Data/NVIDIA Corporation/GeForceNOW/CEF/GeForceNOW.exe
Initial process has started with pid 17486
Game is considered started.
info:  Game: GeForceNOW.exe
info:  DXVK: v1.7.2-5-g13052d49
info:  Built-in extension providers:
info:    Win32 WSI
info:    OpenVR
warn:  OpenVR: Failed to locate module
info:  Enabled instance extensions:
info:    VK_KHR_get_surface_capabilities2
info:    VK_KHR_surface
info:    VK_KHR_win32_surface
skipping config: /home/simon/.config/MangoHud/wine-GeForceNOW.conf [ not found ]
skipping config: /home/simon/.local/share/lutris/runners/wine/lutris-gfn-6-x86_64/bin/MangoHud.conf [ not found ]
skipping config: /home/simon/.config/MangoHud/wine64-preloader.conf [ not found ]
parsing config: /home/simon/.config/MangoHud/MangoHud.conf [ ok ]
info:  AMD RADV POLARIS10 (ACO):
info:    Driver: 20.2.2
info:    Vulkan: 1.2.131
info:    Memory Heap[0]: 
info:      Size: 7936 MiB
info:      Flags: 0x1
info:      Memory Type[0]: Property Flags = 0x1
info:    Memory Heap[1]: 
info:      Size: 8192 MiB
info:      Flags: 0x0
info:      Memory Type[1]: Property Flags = 0x6
info:      Memory Type[3]: Property Flags = 0xe
info:    Memory Heap[2]: 
info:      Size: 256 MiB
info:      Flags: 0x1
info:      Memory Type[2]: Property Flags = 0x7
info:  D3D11CoreCreateDevice: Probing 49152
info:  D3D11CoreCreateDevice: Probing D3D_FEATURE_LEVEL_11_1
info:  D3D11CoreCreateDevice: Using feature level D3D_FEATURE_LEVEL_11_1
info:  Device properties:
info:    Device name:     : AMD RADV POLARIS10 (ACO)
info:    Driver version   : 20.2.2
info:  Enabled device extensions:
info:    VK_AMD_memory_overallocation_behavior
info:    VK_AMD_shader_fragment_mask
info:    VK_EXT_custom_border_color
info:    VK_EXT_depth_clip_enable
info:    VK_EXT_host_query_reset
info:    VK_EXT_memory_priority
info:    VK_EXT_robustness2
info:    VK_EXT_shader_demote_to_helper_invocation
info:    VK_EXT_shader_stencil_export
info:    VK_EXT_shader_viewport_index_layer
info:    VK_EXT_transform_feedback
info:    VK_EXT_vertex_attribute_divisor
info:    VK_KHR_create_renderpass2
info:    VK_KHR_depth_stencil_resolve
info:    VK_KHR_draw_indirect_count
info:    VK_KHR_driver_properties
info:    VK_KHR_image_format_list
info:    VK_KHR_sampler_mirror_clamp_to_edge
info:    VK_KHR_swapchain
info:  Device features:
info:    robustBufferAccess                     : 1
info:    fullDrawIndexUint32                    : 1
info:    imageCubeArray                         : 1
info:    independentBlend                       : 1
info:    geometryShader                         : 1
info:    tessellationShader                     : 1
info:    sampleRateShading                      : 1
info:    dualSrcBlend                           : 1
info:    logicOp                                : 1
info:    multiDrawIndirect                      : 1
info:    drawIndirectFirstInstance              : 1
info:    depthClamp                             : 1
info:    depthBiasClamp                         : 1
info:    fillModeNonSolid                       : 1
info:    depthBounds                            : 1
info:    multiViewport                          : 1
info:    samplerAnisotropy                      : 1
info:    textureCompressionBC                   : 1
info:    occlusionQueryPrecise                  : 1
info:    pipelineStatisticsQuery                : 1
info:    vertexPipelineStoresAndAtomics         : 1
info:    fragmentStoresAndAtomics               : 1
info:    shaderImageGatherExtended              : 1
info:    shaderStorageImageExtendedFormats      : 1
info:    shaderStorageImageReadWithoutFormat    : 1
info:    shaderStorageImageWriteWithoutFormat   : 1
info:    shaderClipDistance                     : 1
info:    shaderCullDistance                     : 1
info:    shaderFloat64                          : 1
info:    shaderInt64                            : 1
info:    variableMultisampleRate                : 1
info:  VK_EXT_conditional_rendering
info:    conditionalRendering                   : 1
info:  VK_EXT_4444_formats
info:    formatA4R4G4B4                         : 0
info:    formatA4B4G4R4                         : 0
info:  VK_EXT_custom_border_color
info:    customBorderColors                     : 1
info:    customBorderColorWithoutFormat         : 1
info:  VK_EXT_depth_clip_enable
info:    depthClipEnable                        : 1
info:  VK_EXT_extended_dynamic_state
info:    extendedDynamicState                   : 0
info:  VK_EXT_host_query_reset
info:    hostQueryReset                         : 1
info:  VK_EXT_memory_priority
info:    memoryPriority                         : 1
info:  VK_EXT_robustness2
info:    robustBufferAccess2                    : 1
info:    robustImageAccess2                     : 1
info:    nullDescriptor                         : 1
info:  VK_EXT_shader_demote_to_helper_invocation
info:    shaderDemoteToHelperInvocation         : 1
info:  VK_EXT_transform_feedback
info:    transformFeedback                      : 1
info:    geometryStreams                        : 1
info:  VK_EXT_vertex_attribute_divisor
info:    vertexAttributeInstanceRateDivisor     : 1
info:    vertexAttributeInstanceRateZeroDivisor : 1
info:  Queue families:
info:    Graphics : 0
info:    Transfer : 1
MANGOHUD: can't find file pci.ids
info:  DXVK: Read 0 valid state cache entries
info:  DXVK: Using 2 compiler threads
warn:  DXGI: VK_FORMAT_D24_UNORM_S8_UINT -> VK_FORMAT_D32_SFLOAT_S8_UINT
info:  Adapter LUID 0: 0:3f8
err:   D3D11DXGIResource::CreateSharedHandle: Stub
wine: Read access denied for device L"\\??\\E:\\", FS volume label and serial are not available.
wine: Read access denied for device L"\\??\\E:\\", FS volume label and serial are not available.
wine: Read access denied for device L"\\??\\E:\\", FS volume label and serial are not available.
wine: Read access denied for device L"\\??\\E:\\", FS volume label and serial are not available.
wine: Read access denied for device L"\\??\\E:\\", FS volume label and serial are not available.
wine: Read access denied for device L"\\??\\E:\\", FS volume label and serial are not available.
wine: Read access denied for device L"\\??\\E:\\", FS volume label and serial are not available.
wine: Read access denied for device L"\\??\\E:\\", FS volume label and serial are not available.
[3920-11-07/ 18:04:28.750:ERROR:network_change_notifier_win.cc(142)] WSALookupServiceBegin failed with: 0
[3920-11-07/ 18:04:28.817:INFO:browser_app.cc(408)] Framework Microsoft Windows 64bit
[3920-11-07/ 18:04:28.817:INFO:browser_app.cc(412)] Version NVIDIACEFClient/84.4147.89.1/gfn_production/e1eecdcad
[3920-11-07/ 18:04:28.818:INFO:browser_app.cc(742)] No urlRoute value found.
[3920-11-07/ 18:04:28.821:INFO:simple_grid_app.cc(52)] Creating SimpleGridApp
[3920-11-07/ 18:04:28.860:INFO:cef_plugin_win.cpp(28)] Loaded dependency module CrimsonUtil.dll
[3920-11-07/ 18:04:29.046:INFO:rtsp_handler_win.cc(69)] Serenity: RTSP interface is ready for ShareServer.dll
[3920-11-07/ 18:04:29.076:INFO:rtsp_handler_win.cc(69)] Serenity: RTSP interface is ready for gfnspfbc.dll
[3920-11-07/ 18:04:29.127:INFO:rtsp_handler_win.cc(69)] Serenity: RTSP interface is ready for NvCamera.dll
[3920-11-07/ 18:04:29.127:INFO:browser_app.cc(834)] Geronimo Logger Init
[3920-11-07/ 18:04:29.128:INFO:geronimo_logger.cpp(20)] Creating GeronimoLogger
[3920-11-07/ 18:04:29.128:INFO:browser_app.cc(839)] shared storage path for geronimo C:\users\simon\Local Settings\Application Data\NVIDIA Corporation\GeForceNOW\sharedstorage.json
[3920-11-07/ 18:04:29.129:INFO:browser_app.cc(845)] Geronimo SDL Init
[3920-11-07/ 18:04:29.134:INFO:geronimo_analytics.cpp(16)] Creating GeronimoDispatcher
[3920-11-07/ 18:04:29.134:INFO:browser_app.cc(859)] Geronimo Tracing Init
[3920-11-07/ 18:04:29.134:INFO:browser_app.cc(862)] Geronimo GRID Init
I-!#>
 [2020-11-07 17:04:29,137]=18:04:29=    INFO [Common: Network] {000000BC} - BCryptGenRandom succeeded
I-!#>
 [2020-11-07 17:04:29,137]=18:04:29=    INFO [Common: Network] {000000BC} - Successfully set overridden random method
I-!$> [2020-11-07 17:04:29,137]=18:04:29=    INFO [BifrostClient: Scheduler] {000001E4} - Successfully set TLS with value 0000000000BCAC90
I-!$> [2020-11-07 17:04:29,138]=18:04:29=   DEBUG [BifrostClient: Interface] {000000BC} - nvbCreateClient( 0000000000B3D550 )
[3920-11-07/ 18:04:29.137:INFO:browser_app.cc(872)] Grid interface is 0000000000C2FE10
[3920-11-07/ 18:04:29.137:INFO:offscreen_sdlwindow_mgr.cpp(30)] Creating SWindowManager
[3920-11-07/ 18:04:29.137:INFO:offscreen_sdlwindow_mgr.cpp(43)] Before SWindowManager::init(), m_eprocessor is 0000000000000000
info:  Game: GeForceNOW.exe
info:  DXVK: v1.7.2-5-g13052d49
info:  Built-in extension providers:
info:    Win32 WSI
info:    OpenVR
warn:  OpenVR: Failed to locate module
info:  Enabled instance extensions:
info:    VK_KHR_get_surface_capabilities2
info:    VK_KHR_surface
info:    VK_KHR_win32_surface
skipping config: /home/simon/.config/MangoHud/wine-GeForceNOW.conf [ not found ]
skipping config: /home/simon/.local/share/lutris/runners/wine/lutris-gfn-6-x86_64/bin/MangoHud.conf [ not found ]
skipping config: /home/simon/.config/MangoHud/wine64-preloader.conf [ not found ]
parsing config: /home/simon/.config/MangoHud/MangoHud.conf [ ok ]
info:  AMD RADV POLARIS10 (ACO):
info:    Driver: 20.2.2
info:    Vulkan: 1.2.131
info:    Memory Heap[0]: 
info:      Size: 7936 MiB
info:      Flags: 0x1
info:      Memory Type[0]: Property Flags = 0x1
info:    Memory Heap[1]: 
info:      Size: 8192 MiB
info:      Flags: 0x0
info:      Memory Type[1]: Property Flags = 0x6
info:      Memory Type[3]: Property Flags = 0xe
info:    Memory Heap[2]: 
info:      Size: 256 MiB
info:      Flags: 0x1
info:      Memory Type[2]: Property Flags = 0x7
info:  D3D11CoreCreateDevice: Probing D3D_FEATURE_LEVEL_11_1
info:  D3D11CoreCreateDevice: Using feature level D3D_FEATURE_LEVEL_11_1
info:  Device properties:
info:    Device name:     : AMD RADV POLARIS10 (ACO)
info:    Driver version   : 20.2.2
info:  Enabled device extensions:
info:    VK_AMD_memory_overallocation_behavior
info:    VK_AMD_shader_fragment_mask
info:    VK_EXT_custom_border_color
info:    VK_EXT_depth_clip_enable
info:    VK_EXT_host_query_reset
info:    VK_EXT_memory_priority
info:    VK_EXT_robustness2
info:    VK_EXT_shader_demote_to_helper_invocation
info:    VK_EXT_shader_stencil_export
info:    VK_EXT_shader_viewport_index_layer
info:    VK_EXT_transform_feedback
info:    VK_EXT_vertex_attribute_divisor
info:    VK_KHR_create_renderpass2
info:    VK_KHR_depth_stencil_resolve
info:    VK_KHR_draw_indirect_count
info:    VK_KHR_driver_properties
info:    VK_KHR_image_format_list
info:    VK_KHR_sampler_mirror_clamp_to_edge
info:    VK_KHR_swapchain
info:  Device features:
info:    robustBufferAccess                     : 1
info:    fullDrawIndexUint32                    : 1
info:    imageCubeArray                         : 1
info:    independentBlend                       : 1
info:    geometryShader                         : 1
info:    tessellationShader                     : 1
info:    sampleRateShading                      : 1
info:    dualSrcBlend                           : 1
info:    logicOp                                : 1
info:    multiDrawIndirect                      : 1
info:    drawIndirectFirstInstance              : 1
info:    depthClamp                             : 1
info:    depthBiasClamp                         : 1
info:    fillModeNonSolid                       : 1
info:    depthBounds                            : 1
info:    multiViewport                          : 1
info:    samplerAnisotropy                      : 1
info:    textureCompressionBC                   : 1
info:    occlusionQueryPrecise                  : 1
info:    pipelineStatisticsQuery                : 1
info:    vertexPipelineStoresAndAtomics         : 1
info:    fragmentStoresAndAtomics               : 1
info:    shaderImageGatherExtended              : 1
info:    shaderStorageImageExtendedFormats      : 1
info:    shaderStorageImageReadWithoutFormat    : 1
info:    shaderStorageImageWriteWithoutFormat   : 1
info:    shaderClipDistance                     : 1
info:    shaderCullDistance                     : 1
info:    shaderFloat64                          : 1
info:    shaderInt64                            : 1
info:    variableMultisampleRate                : 1
info:  VK_EXT_conditional_rendering
info:    conditionalRendering                   : 1
info:  VK_EXT_4444_formats
info:    formatA4R4G4B4                         : 0
info:    formatA4B4G4R4                         : 0
info:  VK_EXT_custom_border_color
info:    customBorderColors                     : 1
info:    customBorderColorWithoutFormat         : 1
info:  VK_EXT_depth_clip_enable
info:    depthClipEnable                        : 1
info:  VK_EXT_extended_dynamic_state
info:    extendedDynamicState                   : 0
info:  VK_EXT_host_query_reset
info:    hostQueryReset                         : 1
info:  VK_EXT_memory_priority
info:    memoryPriority                         : 1
info:  VK_EXT_robustness2
info:    robustBufferAccess2                    : 1
info:    robustImageAccess2                     : 1
info:    nullDescriptor                         : 1
info:  VK_EXT_shader_demote_to_helper_invocation
info:    shaderDemoteToHelperInvocation         : 1
info:  VK_EXT_transform_feedback
info:    transformFeedback                      : 1
info:    geometryStreams                        : 1
info:  VK_EXT_vertex_attribute_divisor
info:    vertexAttributeInstanceRateDivisor     : 1
info:    vertexAttributeInstanceRateZeroDivisor : 1
info:  Queue families:
info:    Graphics : 0
info:    Transfer : 1
MANGOHUD: can't find file pci.ids
info:  Game: GeForceNOW.exe
info:  DXVK: v1.7.2-5-g13052d49
info:  Built-in extension providers:
info:    Win32 WSI
info:    OpenVR
warn:  OpenVR: Failed to locate module
info:  Enabled instance extensions:
info:    VK_KHR_get_surface_capabilities2
info:    VK_KHR_surface
info:    VK_KHR_win32_surface
skipping config: /home/simon/.config/MangoHud/wine-GeForceNOW.conf [ not found ]
skipping config: /home/simon/.local/share/lutris/runners/wine/lutris-gfn-6-x86_64/bin/MangoHud.conf [ not found ]
skipping config: /home/simon/.config/MangoHud/wine64-preloader.conf [ not found ]
parsing config: /home/simon/.config/MangoHud/MangoHud.conf [ ok ]
info:  AMD RADV POLARIS10 (ACO):
info:    Driver: 20.2.2
info:    Vulkan: 1.2.131
info:    Memory Heap[0]: 
info:      Size: 7936 MiB
info:      Flags: 0x1
info:      Memory Type[0]: Property Flags = 0x1
info:    Memory Heap[1]: 
info:      Size: 8192 MiB
info:      Flags: 0x0
info:      Memory Type[1]: Property Flags = 0x6
info:      Memory Type[3]: Property Flags = 0xe
info:    Memory Heap[2]: 
info:      Size: 256 MiB
info:      Flags: 0x1
info:      Memory Type[2]: Property Flags = 0x7
info:  Adapter LUID 0: 0:3fa
info:  DXVK: Read 0 valid state cache entries
info:  DXVK: Using 2 compiler threads
warn:  DXGI: VK_FORMAT_D24_UNORM_S8_UINT -> VK_FORMAT_D32_SFLOAT_S8_UINT
info:  Game: GeForceNOW.exe
info:  DXVK: v1.7.2-5-g13052d49
info:  Built-in extension providers:
info:    Win32 WSI
info:    OpenVR
warn:  OpenVR: Failed to locate module
info:  Enabled instance extensions:
info:    VK_KHR_get_surface_capabilities2
info:    VK_KHR_surface
info:    VK_KHR_win32_surface
skipping config: /home/simon/.config/MangoHud/wine-GeForceNOW.conf [ not found ]
skipping config: /home/simon/.local/share/lutris/runners/wine/lutris-gfn-6-x86_64/bin/MangoHud.conf [ not found ]
skipping config: /home/simon/.config/MangoHud/wine64-preloader.conf [ not found ]
parsing config: /home/simon/.config/MangoHud/MangoHud.conf [ ok ]
warn:  D3D9: VK_FORMAT_D24_UNORM_S8_UINT -> VK_FORMAT_D32_SFLOAT_S8_UINT
info:  AMD RADV POLARIS10 (ACO):
info:    Driver: 20.2.2
info:    Vulkan: 1.2.131
info:    Memory Heap[0]: 
info:      Size: 7936 MiB
info:      Flags: 0x1
info:      Memory Type[0]: Property Flags = 0x1
info:    Memory Heap[1]: 
info:      Size: 8192 MiB
info:      Flags: 0x0
info:      Memory Type[1]: Property Flags = 0x6
info:      Memory Type[3]: Property Flags = 0xe
info:    Memory Heap[2]: 
info:      Size: 256 MiB
info:      Flags: 0x1
info:      Memory Type[2]: Property Flags = 0x7
info:  Process set as DPI aware
[3920-11-07/ 18:04:29.633:ERROR:dxva_video_decode_accelerator_win.cc(1431)] DXVAVDA fatal error: could not LoadLibrary: msmpeg2vdec.dll: Module not found. (0x7E)
[3920-11-07/ 18:04:29.681:INFO:offscreen_sdlwindow_mgr.cpp(105)] Init SDL Event Processor, m_eprocessor is 0000000010128790
[3920-11-07/ 18:04:29.681:INFO:offscreen_sdlwindow_mgr.cpp(125)] In exit of SWindowManager::init() m_eprocessor is 0000000010128790
[3920-11-07/ 18:04:29.681:INFO:browser_app.cc(58)] Creating SGamepad
[3920-11-07/ 18:04:29.689:INFO:cef_plugin_manager.cpp(25)] Creating CefPluginManager
[3920-11-07/ 18:04:29.689:INFO:browser_app.cc(995)] CreateCefWindowInstance
[3920-11-07/ 18:04:29.689:INFO:query_handler.cc(33)] Creating QueryHandler
[3920-11-07/ 18:04:29.690:INFO:osr_query_handler.cc(21)] Creating OSRQueryHandler
[3920-11-07/ 18:04:29.690:INFO:gfn_query_handler.cc(48)] Creating GFNQueryHandler
[3920-11-07/ 18:04:29.690:INFO:simple_handler.cc(43)] Creating SimpleHandler
[3920-11-07/ 18:04:29.690:INFO:osr_simple_handler.cpp(37)] Creating OSRSimpleHandler
[3920-11-07/ 18:04:29.690:INFO:browser_app.cc(999)] Add handler 000000001013E240
[3920-11-07/ 18:04:29.691:INFO:cef_window.cpp(59)] Creating cef_window
[3920-11-07/ 18:04:29.691:INFO:cef_dark_window.cpp(191)] Creating cef_dark_window
[3920-11-07/ 18:04:29.691:INFO:cef_dark_aero_window.cpp(203)] Creating cef_dark_aero_window
[3920-11-07/ 18:04:29.692:INFO:cef_dark_window.cpp(244)] GDI+ startup
[3920-11-07/ 18:04:29.729:INFO:cef_dark_window.cpp(827)] Could not get browser reference while drawing system icons
[3920-11-07/ 18:04:29.732:INFO:cef_dark_window.cpp(827)] Could not get browser reference while drawing system icons
[3920-11-07/ 18:04:29.738:INFO:cef_dark_window.cpp(827)] Could not get browser reference while drawing system icons
[3920-11-07/ 18:04:29.743:INFO:cef_dark_window.cpp(827)] Could not get browser reference while drawing system icons
[3920-11-07/ 18:04:29.746:INFO:cef_dark_window.cpp(827)] Could not get browser reference while drawing system icons
[3920-11-07/ 18:04:29.765:INFO:offscreen_sdlwindow.cpp(44)] Creating SWindow
[3920-11-07/ 18:04:29.765:INFO:offscreen_sdlwindow_mgr.cpp(179)] SDLWindowMgr createWindow callback for 000000001018F6C0
[3920-11-07/ 18:04:29.771:INFO:cef_dark_window.cpp(827)] Could not get browser reference while drawing system icons
warn:  winevulkan detected, disabling exclusive fullscreen support
info:  Presenter: Actual swap chain properties:
info:    Format:       VK_FORMAT_B8G8R8A8_UNORM
info:    Present mode: VK_PRESENT_MODE_IMMEDIATE_KHR
info:    Buffer size:  1240x773
info:    Image count:  3
info:    Exclusive FS: 2
warn:  [3920-11-07/ 18:04:29.786:INFO:simple_grid_app.cc(797)] sendAnalyticsEvent(Debug, , )
[3920-11-07/ 18:04:29.786:INFO:simple_grid_app.cc(315)] No GridApp callback found
DXGI: MakeWindowAssociation: Ignoring flags
info:  Presenter: Actual swap chain properties:
info:    Format:       VK_FORMAT_B8G8R8A8_UNORM
info:    Present mode: VK_PRESENT_MODE_FIFO_KHR
info:    Buffer size:  1240x773
info:    Image count:  4
info:    Exclusive FS: 2
[3920-11-07/ 18:04:29.792:INFO:osr_simple_handler.cpp(90)] Add window 000000001018F6C0
[3920-11-07/ 18:04:29.794:INFO:cef_dark_window.cpp(827)] Could not get browser reference while drawing system icons
[3920-11-07/ 18:04:29.801:INFO:cef_dark_window.cpp(827)] Could not get browser reference while drawing system icons
[3920-11-07/ 18:04:29.805:INFO:cef_dark_window.cpp(827)] Could not get browser reference while drawing system icons
[3920-11-07/ 18:04:29.809:INFO:offscreen_window.cpp(61)] Creating OSRWindow
info:  Game: GeForceNOW.exe
info:  DXVK: v1.7.2-5-g13052d49
info:  Built-in extension providers:
info:    Win32 WSI
info:    OpenVR
warn:  OpenVR: Failed to locate module
info:  Enabled instance extensions:
info:    VK_KHR_get_surface_capabilities2
info:    VK_KHR_surface
info:    VK_KHR_win32_surface
skipping config: /home/simon/.config/MangoHud/wine-GeForceNOW.conf [ not found ]
skipping config: /home/simon/.local/share/lutris/runners/wine/lutris-gfn-6-x86_64/bin/MangoHud.conf [ not found ]
skipping config: /home/simon/.config/MangoHud/wine64-preloader.conf [ not found ]
parsing config: /home/simon/.config/MangoHud/MangoHud.conf [ ok ]
info:  AMD RADV POLARIS10 (ACO):
info:    Driver: 20.2.2
info:    Vulkan: 1.2.131
info:    Memory Heap[0]: 
info:      Size: 7936 MiB
info:      Flags: 0x1
info:      Memory Type[0]: Property Flags = 0x1
info:    Memory Heap[1]: 
info:      Size: 8192 MiB
info:      Flags: 0x0
info:      Memory Type[1]: Property Flags = 0x6
info:      Memory Type[3]: Property Flags = 0xe
info:    Memory Heap[2]: 
info:      Size: 256 MiB
info:      Flags: 0x1
info:      Memory Type[2]: Property Flags = 0x7
info:  D3D11CoreCreateDevice: Probing 49152
info:  D3D11CoreCreateDevice: Probing D3D_FEATURE_LEVEL_11_1
info:  D3D11CoreCreateDevice: Using feature level D3D_FEATURE_LEVEL_11_1
info:  Device properties:
info:    Device name:     : AMD RADV POLARIS10 (ACO)
info:    Driver version   : 20.2.2
info:  Enabled device extensions:
info:    VK_AMD_memory_overallocation_behavior
info:    VK_AMD_shader_fragment_mask
info:    VK_EXT_custom_border_color
info:    VK_EXT_depth_clip_enable
info:    VK_EXT_host_query_reset
info:    VK_EXT_memory_priority
info:    VK_EXT_robustness2
info:    VK_EXT_shader_demote_to_helper_invocation
info:    VK_EXT_shader_stencil_export
info:    VK_EXT_shader_viewport_index_layer
info:    VK_EXT_transform_feedback
info:    VK_EXT_vertex_attribute_divisor
info:    VK_KHR_create_renderpass2
info:    VK_KHR_depth_stencil_resolve
info:    VK_KHR_draw_indirect_count
info:    VK_KHR_driver_properties
info:    VK_KHR_image_format_list
info:    VK_KHR_sampler_mirror_clamp_to_edge
info:    VK_KHR_swapchain
info:  Device features:
info:    robustBufferAccess                     : 1
info:    fullDrawIndexUint32                    : 1
info:    imageCubeArray                         : 1
info:    independentBlend                       : 1
info:    geometryShader                         : 1
info:    tessellationShader                     : 1
info:    sampleRateShading                      : 1
info:    dualSrcBlend                           : 1
info:    logicOp                                : 1
info:    multiDrawIndirect                      : 1
info:    drawIndirectFirstInstance              : 1
info:    depthClamp                             : 1
info:    depthBiasClamp                         : 1
info:    fillModeNonSolid                       : 1
info:    depthBounds                            : 1
info:    multiViewport                          : 1
info:    samplerAnisotropy                      : 1
info:    textureCompressionBC                   : 1
info:    occlusionQueryPrecise                  : 1
info:    pipelineStatisticsQuery                : 1
info:    vertexPipelineStoresAndAtomics         : 1
info:    fragmentStoresAndAtomics               : 1
info:    shaderImageGatherExtended              : 1
info:    shaderStorageImageExtendedFormats      : 1
info:    shaderStorageImageReadWithoutFormat    : 1
info:    shaderStorageImageWriteWithoutFormat   : 1
info:    shaderClipDistance                     : 1
info:    shaderCullDistance                     : 1
info:    shaderFloat64                          : 1
info:    shaderInt64                            : 1
info:    variableMultisampleRate                : 1
info:  VK_EXT_conditional_rendering
info:    conditionalRendering                   : 1
info:  VK_EXT_4444_formats
info:    formatA4R4G4B4                         : 0
info:    formatA4B4G4R4                         : 0
info:  VK_EXT_custom_border_color
info:    customBorderColors                     : 1
info:    customBorderColorWithoutFormat         : 1
info:  VK_EXT_depth_clip_enable
info:    depthClipEnable                        : 1
info:  VK_EXT_extended_dynamic_state
info:    extendedDynamicState                   : 0
info:  VK_EXT_host_query_reset
info:    hostQueryReset                         : 1
info:  VK_EXT_memory_priority
info:    memoryPriority                         : 1
info:  VK_EXT_robustness2
info:    robustBufferAccess2                    : 1
info:    robustImageAccess2                     : 1
info:    nullDescriptor                         : 1
info:  VK_EXT_shader_demote_to_helper_invocation
info:    shaderDemoteToHelperInvocation         : 1
info:  VK_EXT_transform_feedback
info:    transformFeedback                      : 1
info:    geometryStreams                        : 1
info:  VK_EXT_vertex_attribute_divisor
info:    vertexAttributeInstanceRateDivisor     : 1
info:    vertexAttributeInstanceRateZeroDivisor : 1
info:  Queue families:
info:    Graphics : 0
info:    Transfer : 1
MANGOHUD: can't find file pci.ids
info:  DXVK: Read 0 valid state cache entries
info:  DXVK: Using 2 compiler threads
warn:  DXGI: VK_FORMAT_D24_UNORM_S8_UINT -> VK_FORMAT_D32_SFLOAT_S8_UINT
err:   D3D11DXGIResource::CreateSharedHandle: Stub
[3920-11-07/ 18:04:30.043:WARNING:offscreen_windowing_d3d11.cpp(259)] Couldn't create shared handle for test texture
[3920-11-07/ 18:04:30.122:WARNING:offscreen_window.cpp(70)] disabling gpu
[3920-11-07/ 18:04:30.122:INFO:offscreen_window.cpp(459)] OnSize w=1240 h=773 r=1
[3920-11-07/ 18:04:30.122:INFO:offscreen_window.cpp(415)] setting zoom to: 0
[3920-11-07/ 18:04:30.122:INFO:offscreen_renderer_d3d11.cpp(631)] Initializing osc d3d11 renderer to: 1240 773
[3920-11-07/ 18:04:30.126:INFO:cef_plugin_manager.cpp(171)] Loading all plugins
[3920-11-07/ 18:04:30.127:INFO:cefmain_win.cc(461)] CEF resource load time : 609ms
[3920-11-07/ 18:04:30.127:INFO:browser_app.cc(1083)] Start app event thread
[3920-11-07/ 18:04:30.136:INFO:ipc_api.cc(21)] Creating IpcApi
[3920-11-07/ 18:04:30.136:INFO:ipc_cef.cc(52)] Creating IPC instance with name = GeForceNOW, type = mallclient
[3920-11-07/ 18:04:30.136:INFO:json_storage_win.cc(33)] Lock opened an existing mutex so waiting on NvCrimson.ipcmessages.json
[3920-11-07/ 18:04:30.136:INFO:json_storage_win.cc(35)] Lock mutex aquired
[3920-11-07/ 18:04:30.136:INFO:ipc_cef.cc(74)] Finished initialization.
[3920-11-07/ 18:04:30.137:INFO:osr_simple_handler.cpp(567)] SDL Foreground: Window is not minimized, bringing to front
[3920-11-07/ 18:04:30.137:WARNING:offscreen_layer.cpp(680)] creating browser layer: 1
[3920-11-07/ 18:04:30.137:INFO:file_watcher_win.cc(344)] Detected file modified
[3920-11-07/ 18:04:30.137:INFO:json_storage_win.cc(33)] Lock opened an existing mutex so waiting on NvCrimson.ipcmessages.json
[3920-11-07/ 18:04:30.137:INFO:json_storage_win.cc(35)] Lock mutex aquired
[3920-11-07/ 18:04:30.137:INFO:file_watcher_win.cc(344)] Detected file modified
[3920-11-07/ 18:04:30.137:INFO:json_storage_win.cc(33)] Lock opened an existing mutex so waiting on NvCrimson.ipcmessages.json
[3920-11-07/ 18:04:30.137:INFO:json_storage_win.cc(35)] Lock mutex aquired
wine: Unhandled page fault on execute access to 0000000000000000 at address 0000000000000000 (thread 00bc), starting debugger...
[3920-11-07/ 18:04:30.169:INFO:cef_plugin_win.cpp(28)] Loaded dependency module discord-rpc.dll
[3920-11-07/ 18:04:30.200:INFO:cef_plugin_win.cpp(137)] Loaded Cef Plugin  cef/client/Discord.dll
Unhandled exception: page fault on execute access to 0x00000000 in 64-bit code (0x0000000000000000).
Register dump:
 rip:0000000000000000 rsp:0000000000237e48 rbp:0000000000000005 eflags:00010202 (  R- --  I   - - - )
 rax:00007f717bcdde00 rbx:0000000000000000 rcx:0000000010260690 rdx:0000000010261110
 rsi:0000000008bd0660 rdi:0000000010260180  r8:0000000000000000  r9:0000000000238090 r10:0000000010260480
 r11:0000000010260480 r12:000000000476e5c0 r13:000000001025f620 r14:00000000002382a8 r15:0000000000238090
Stack dump:
0x0000000000237e48:  000000000476e650 0000000000000000
0x0000000000237e58:  0000000000000000 0000000000000000
0x0000000000237e68:  0000000000000000 0000000000000000
0x0000000000237e78:  0000000000000000 0000000000000000
0x0000000000237e88:  0000000000000000 0000000000000000
0x0000000000237e98:  0000000000000000 ffff000000000000
0x0000000000237ea8:  ffffffffffffffff 0000000000000000
0x0000000000237eb8:  0000000000000000 0000000000000000
0x0000000000237ec8:  0000000000000000 0000000000000000
0x0000000000237ed8:  0000000000000000 0000000000000000
0x0000000000237ee8:  0000000000000000 0000000010261110
0x0000000000237ef8:  0000000010260690 0000000000000000
Backtrace:
=>0 0x0000000000000000 (0x0000000000000005)
0x0000000000000000: -- no code accessible --
Modules:
Module	Address					Debug info	Name (105 modules)
PE	          240000-          253000	Deferred        aclui
PE	          7e0000-          7e8000	Deferred        api-ms-win-core-localization-l1-2-1
PE	          c30000-          dca000	Deferred        gsaudiowebrtc
PE	          dd0000-         180e000	Deferred        bifrost2
PE	         1810000-         181b000	Deferred        qwave
PE	         1820000-         1f73000	Deferred        geronimo
PE	         2090000-         218c000	Deferred        chrome_elf
PE	         2190000-         9c69000	Deferred        libcef
PE	         9c80000-         9cd1000	Deferred        oleacc
PE	         9ce0000-         9d07000	Deferred        credui
PE	         9d10000-         9d23000	Deferred        esent
PE	         9d30000-         a15e000	Deferred        d3dcompiler_47
PE	         a160000-         a1c4000	Deferred        libegl
PE	         a1d0000-         af8b000	Deferred        libglesv2
PE	         deb0000-         e18d000	Deferred        crimsonutil
PE	         e190000-         e506000	Deferred        gfnspfbc
PE	         e640000-         e987000	Deferred        gfncapture
PE	         eac0000-         f177000	Deferred        shareserver
PE	         f180000-         f750000	Deferred        nvcamera
PE	        13280000-        1331c000	Deferred        discord-rpc
PE	        13750000-        13a21000	Deferred        discord
PE	        61340000-        61347000	Deferred        api-ms-win-core-localization-obsolete-l1-2-0
PE	        61480000-        6148c000	Deferred        wlanapi
PE	        61900000-        61929000	Deferred        sechost
PE	        61f40000-        61f4e000	Deferred        wtsapi32
PE	        62240000-        624d1000	Deferred        d3d9
PE	        62540000-        62567000	Deferred        wintrust
PE	        62600000-        6263c000	Deferred        usp10
PE	        62740000-        6274f000	Deferred        imagehlp
PE	        628c0000-        628df000	Deferred        uiautomationcore
PE	        637c0000-        63878000	Deferred        winmm
PE	        638c0000-        638ce000	Deferred        dwmapi
PE	        64340000-        64381000	Deferred        winhttp
PE	        64940000-        64959000	Deferred        shcore
PE	        64b80000-        64b8f000	Deferred        hid
PE	        64d40000-        64d4f000	Deferred        vulkan-1
PE	        65000000-        65142000	Deferred        ole32
PE	        65680000-        65688000	Deferred        api-ms-win-core-sysinfo-l1-2-1
PE	        65780000-        6578d000	Deferred        version
PE	        66200000-        6626a000	Deferred        dbghelp
PE	        66500000-        66532000	Deferred        rsaenh
PE	        66600000-        66607000	Deferred        api-ms-win-core-datetime-l1-1-1
PE	        66780000-        66787000	Deferred        api-ms-win-core-string-l1-1-0
PE	        67880000-        67890000	Deferred        wpc
PE	        67d00000-        67d0e000	Deferred        userenv
PE	        67dc0000-        67dd0000	Deferred        cryptnet
PE	        68140000-        68174000	Deferred        msctf
PE	        68a40000-        68a91000	Deferred        shlwapi
PE	        68f80000-        68f92000	Deferred        pdh
PE	        69840000-        6984e000	Deferred        xinput1_4
PE	        69c80000-        69caa000	Deferred        propsys
PE	        6a100000-        6a235000	Deferred        oleaut32
PE	        6a340000-        6a609000	Deferred        d3d11
PE	        6a700000-        6a773000	Deferred        setupapi
PE	        6b2c0000-        6b33c000	Deferred        wininet
PE	        6b6c0000-        6b7bf000	Deferred        comdlg32
PE	        6b880000-        6b887000	Deferred        api-ms-win-core-fibers-l1-1-1
PE	        6bd00000-        6bd0d000	Deferred        schannel
PE	        6bec0000-        6bed8000	Deferred        imm32
PE	        6c100000-        6c108000	Deferred        api-ms-win-appmodel-runtime-l1-1-2
PE	        6c600000-        6c607000	Deferred        api-ms-win-security-systemfunctions-l1-1-0
PE	        6d3c0000-        6d418000	Deferred        dsound
PE	        6d9c0000-        6d9df000	Deferred        mpr
PE	        6dcc0000-        6ddc6000	Deferred        cryptui
PE	        6e280000-        6e28e000	Deferred        ncrypt
PE	        6e340000-        6e348000	Deferred        api-ms-win-core-synch-l1-2-0
PE	        6e6c0000-        6e804000	Deferred        comctl32
PE	        6f200000-        6f3e1000	Deferred        dxgi
PE	        6f480000-        6f505000	Deferred        gdiplus
PE	        6fbc0000-        6fc41000	Deferred        rpcrt4
PE	        70140000-        70165000	Deferred        mmdevapi
PE	        70980000-        709a0000	Deferred        msdmo
PE	        70a00000-        70a0c000	Deferred        dhcpcsvc
PE	        71140000-        711db000	Deferred        urlmon
PE	        71240000-        71260000	Deferred        jsproxy
PE	        7a850000-        7a858000	Deferred        opengl32
PE	        7b000000-        7b0d2000	Deferred        kernelbase
PE	        7b610000-        7b7db000	Deferred        kernel32
PE	        7bc20000-        7bc35000	Deferred        ntdll
PE	       140000000-       140366000	Deferred        geforcenow
PE	       180000000-       18011d000	Deferred        sdl2
PE	    7f7174120000-    7f7174128000	Deferred        windowscodecs
PE	    7f7174220000-    7f717422c000	Deferred        dinput8
PE	    7f717b6c0000-    7f717b6de000	Deferred        wldap32
PE	    7f717baa0000-    7f717baa4000	Deferred        winepulse
PE	    7f717bbd0000-    7f717bbd4000	Deferred        kerberos
PE	    7f717bbf0000-    7f717bbfd000	Deferred        winspool
PE	    7f717bc50000-    7f717bc54000	Deferred        dwrite
PE	    7f717bd00000-    7f717bd06000	Deferred        dnsapi
PE	    7f717bd30000-    7f717bd37000	Deferred        netapi32
PE	    7f717bd70000-    7f717bd74000	Deferred        secur32
PE	    7f717bdc0000-    7f717bdc4000	Deferred        dxva2
PE	    7f717be00000-    7f717be10000	Deferred        msvcrt
PE	    7f717bf30000-    7f717bf37000	Deferred        winevulkan
PE	    7f717bf80000-    7f717bf86000	Deferred        iphlpapi
PE	    7f717c680000-    7f717c685000	Deferred        bcrypt
PE	    7f717c6b0000-    7f717c6f7000	Deferred        crypt32
PE	    7f717c7b0000-    7f717c7b5000	Deferred        uxtheme
PE	    7f717cbc0000-    7f717cbc5000	Deferred        winex11
PE	    7f717cca0000-    7f717d59b000	Deferred        shell32
PE	    7f717d6f0000-    7f717d6f6000	Deferred        ws2_32
PE	    7f717d740000-    7f717d831000	Deferred        user32
PE	    7f717d9c0000-    7f717d9cc000	Deferred        gdi32
PE	    7f717db80000-    7f717db9e000	Deferred        ucrtbase
PE	    7f717dc80000-    7f717dc8b000	Deferred        advapi32
Threads:
process  tid      prio (all id:s are in hex)
00000038 services.exe
	["C:\windows\system32\services.exe"]
	0000003c    0
	00000040    0
	0000004c    0
	0000006c    0
	00000094    0
00000044 plugplay.exe
	[C:\windows\system32\plugplay.exe]
	00000048    0
	00000058    0
	0000005c    0
	00000060    0
	0000007c    0
	00000080    0
	00000084    0
	00000198    0
00000064 winedevice.exe
	[C:\windows\system32\winedevice.exe]
	00000068    0
	00000070    0
	00000074    0
	00000078    0
	00000088    0
0000008c winedevice.exe
	[C:\windows\system32\winedevice.exe]
	00000090    0
	00000098    0
	0000009c    0
	000000a0    0
000000a4 explorer.exe
	[C:\windows\system32\explorer.exe /desktop]
	000000a8    0
	000000ac    0
	000000b0    0
	000000b4    0
000000b8 (D) C:\users\simon\Local Settings\Application Data\NVIDIA Corporation\GeForceNOW\CEF\GeForceNOW.exe
	["C:\users\simon\Local Settings\Application Data\NVIDIA Corporation\GeForceNOW\CEF\GeForceNOW.exe"]
	000000bc    0 <==
	00000140    0
	00000144    0
	00000148   -2
	0000014c    0
	00000150    0
	00000154    0
	00000158    0
	0000015c    0
	00000160    0
	00000164    0
	00000168    0
	0000016c    0
	00000170    0
	00000174    0
	00000178    0
	0000017c    0
	00000180    0
	00000184    0
	00000190    0
	00000194    0
	0000019c   -2
	000001a0    0
	000001a4    0
	000001a8    0
	000001ac    0
	000001c8    0
	000001cc    0
	000001d0    0
	000001d4    0
	000001d8    0
	000001dc    0
	000001e0    0
	000001e4    0
	000001e8    0
	000001f0   -2
	000001f4   -2
	000001f8    0
	000001fc    0
	00000200    0
	00000204    0
	0000020c    0
	00000210    0
	00000214    0
	0000024c    0
	00000250   -2
	00000254    0
	00000258    0
	0000025c    0
	00000260    0
	00000264    0
	00000278    0
000000c0 GeForceNOWReliabilityMonitor.exe
	["C:\users\simon\Local Settings\Application Data\NVIDIA Corporation\GeForceNOW\CEF\GeForceNOWReliabilityMonitor.exe"]
	000000c4    0
	000000d0    0
	000000f0    0
	000000f4    0
	000000f8    0
	000000fc    0
	00000100    0
	00000104    0
	0000010c    0
	00000110    0
	00000114    0
	00000118    0
	0000011c    0
	00000120    0
	00000124    0
	00000128    0
	0000012c    0
	00000130    0
	00000134    0
	00000138    0
	0000013c    0
000000c8 GeForceNOWContainer.exe
	["C:\users\simon\Local Settings\Application Data\NVIDIA Corporation\GeForceNOW\CEF\GeForceNOWContainer.exe" -c -r -d "C:\users\simon\Local Settings\Application Data\NVIDIA Corporation\GeForceNOW\CEF\nvc/localuser" -f "C:\users\simon\Local Settings\Application Data\NVIDIA Corporation\GeForceNOW\logs\NvContainer\GeForceNOWContainer.log" -l 3]
	000000cc    0
	000000d4    0
	000001b0    0
	000001b8    0
	000001bc    0
	000001c0    0
	000001c4    0
00000188 GeForceNOW.exe
	["C:\users\simon\Local Settings\Application Data\NVIDIA Corporation\GeForceNOW\CEF\GeForceNOW.exe" --type=gpu-process --field-trial-handle=676,8784949662054605502,5530534163244386779,131072 --enable-features=CastMediaRouteProvider --disable-features=OutOfBlinkCors --no-sandbox --log-file="C:\users\simon\Local Settings\Application Data\NVIDIA Corporation\GeForceNOW\debug.log" --log-severity=info --lang=en-US --gpu-preferences=MAAAAAAAAADgAAAwAAAAAAAAAAAAAAAAAABgAAAAAAAQAAAAAAAAAAAAAAAAAAAAKAAAAAQAAAAgAAAAAAAAACgAAAAAAAAAMAAAAAAAAAA4AAAAAAAAABAAAAAAAAAAAAAAAAUAAAAQAAAAAAAAAAAAAAAGAAAAEAAAAAAAAAABAAAABQAAABAAAAAAAAAAAQAAAAYAAAA= --use-gl=swiftshader-webgl --log-file="C:\users\simon\Local Settings\Application Data\NVIDIA Corporation\GeForceNOW\debug.log" --mojo-platform-channel-handle=688 /prefetch:2]
	0000018c    1
	000001ec    0
	00000218    0
	0000021c    0
	00000220   -2
	00000224    1
	00000228    0
	0000022c    0
	00000230    1
00000268 GeForceNOW.exe
	["C:\users\simon\Local Settings\Application Data\NVIDIA Corporation\GeForceNOW\CEF\GeForceNOW.exe" --type=renderer --no-sandbox --autoplay-policy=no-user-gesture-required --log-file="C:\users\simon\Local Settings\Application Data\NVIDIA Corporation\GeForceNOW\debug.log" --field-trial-handle=676,8784949662054605502,5530534163244386779,131072 --enable-features=CastMediaRouteProvider --disable-features=OutOfBlinkCors --disable-gpu-compositing --lang=en-US --log-file="C:\users\simon\Local Settings\Application Data\NVIDIA Corporation\GeForceNOW\debug.log" --log-severity=info --device-scale-factor=1 --num-raster-threads=2 --enable-main-frame-before-activation --renderer-client-id=3 --mojo-platform-channel-handle=1636 /prefetch:1]
	0000026c    0
	00000290    0
	00000294    0
	00000298    0
	0000029c   -2
	000002a0    1
	000002a4    0
	000002a8    0
0000027c explorer.exe
	[C:\windows\system32\explorer.exe /desktop]
	00000280    0
	00000284    0
	00000288    0
	0000028c    0
System information:
    Wine build: wine-4.11-7296-g60ed72fc8f (Staging)
    Platform: x86_64
    Version: Windows 7
    Host system: Linux
    Host version: 5.4.60
[3920-11-07/ 18:04:33.680:ERROR:broker_win.cc(55)] Error reading broker pipe: Broken pipe. (0x6D)
Initial process has exited.
2020-11-07T18:04:28.859[I]                     LoggingImpl.cpp:203  Initialize CrimsonNative logger
```

I did copy `msmpeg2vdec.dll` next to the `GeForceNOW.exe` executable which solved the `ERROR:dxva_video_decode_accelerator_win.cc(1431)] DXVAVDA fatal error: could not LoadLibrary: msmpeg2vdec.dll: Module not found. (0x7E)` error but didn't make a difference, the application still crashes the same way, no relevant changes in the log output.

**Comment by byte1024** [comment]
Fixed for me by using winehq staging 5.20. I'm not sure if Lutris has anything new enough without using System wine. But I didn't test others.

About all I can guess in that gforce now installs a latest version on installation and its incompatible with the older selected wine version of the script. Its either that or the script was pushed broken. 

The fix here should be to change the selection wine version.

**Comment by strycore** [comment]
@tannisroot this would be resolved by pushing a release for lutris 5.20

**Comment by strycore** [comment]
not sure, I managed to run the client but then I was unable to run any game as it refused to connect to the Now servers

**Comment by strycore** [comment]
The Web version runs great btw

**Comment by byte1024** [comment]
> not sure, I managed to run the client but then I was unable to run any game as it refused to connect to the Now servers

I didn't test any games, just made sure it didn't crash on start as according to the issue. Suppose, the new wine versions might have regressions if that's what happened there but have seen connection like issues with another application when using anything but winehq staging. Wine builds seem to be a bit to finicky at times.

**Comment by simonvanderveldt** [comment]
Thanks for the info/feedback so far. Unfortunately I have an issue with wine versions > 5.10 so I can't currently test with 5.20/5.21.

The web based installer worked fine but the input lag is horrendous, around a second I think, so it's unusable.

**Comment by databoose** [comment]
> The Web version runs great btw

Runs absolutely horrible for me, i could have times where i would have to wait 6 seconds to see any of my input register at all, whereas in the client version latency was barely even noticable.

**Comment by strycore** [comment]
Weird, I tried a game of Fortnite and it was really playable, even on Wi-fi. Had no connectivity issue compared to a few months ago.

**Comment by simonvanderveldt** [comment]
> Fixed for me by using winehq staging 5.20. I'm not sure if Lutris has anything new enough without using System wine. But I didn't test others.

@byte1024 Did you change anything else? I managed to fix my issues with wine > 5.10 and tested running with wine-staging 5.20 as well as 5.21 but in both cases Lutris still crashes for me. I still need to compare if it's a different crash than with the old 5.6 version provided by Lutris.
And are you using 5.20 from your distro or a Lutris build?

**Comment by byte1024** [comment]
> > Fixed for me by using winehq staging 5.20. I'm not sure if Lutris has anything new enough without using System wine. But I didn't test others.
> 
> @byte1024 Did you change anything else? I managed to fix my issues with wine > 5.10 and tested running with wine-staging 5.20 as well as 5.21 but in both cases Lutris still crashes for me. I still need to compare if it's a different crash than with the old 5.6 version provided by Lutris.
> And are you using 5.20 from your distro or a Lutris build?

No, just installed per the link in the issue, started to crash, changed to wine staging, started without crash.

Using winehq-staging right now which cannot be installed at the same time as distro wine. This is direct install from wine devs: https://wiki.winehq.org/Fedora However, I do install various distro wine provided packages that can be installed at the same time and are required for various games such as wine-mono and all the wine fonts packages, etc.

**Comment by dineiger** [comment]
Just to give you more to work with: I had the same problem (crash on startup with a black window). Per the advice in this thread I installed Wine 5.20 which fixed the problem. I could start the client but when I tried to start any game, it gave me a network connection error. Today it's something else: directly after startup it prompts "Geforce Now needs to be updated" and crashes.

**Comment by byte1024** [comment]
I tested this further.

Yes, latest version fails to connect to the network for Steam, games, etc with recent Wine versions even though it works fine for searching / browsing.

Older Geforce Now application verisons also run into this. However they still work fine with lutris gfn wine as long as they don't update.

That's a bit of a problem. I don't think its a great idea but one might be workaround it for the moment by obtaining an older version and installing it manually. I only tested one restart after it said it would update after that and it didn't but that doesn't mean it won't.

The only good way is to patch Wine I would think. That could mean patching the older Wine or the newer Wine. Not sure if its possible to edit Geforce Now files, could be, but its possibly the worst option.

**Comment by databoose** [comment]
> The only good way is to patch Wine I would think. That could mean patching the older Wine or the newer Wine. Not sure if its possible to edit Geforce Now files, could be, but its possibly the worst option.

If all else fails, the specific patch(s) pertaining to whatever wine code is responsible for breaking gfn compatibility could be reverted to wine 5.20 but still have the networking code of an older version that previously worked with it, and have that as a geforce now specific wine fork, assuming these two things are fairly agnostic to each other and wouldn't require too much rewriting.

For the end user my recommendation would be to not use geforce now altogether and instead opt to use a service like shadow PC cloud streaming or the upcoming amazon luna that looks promising, obviously this is a wine issue but it's also pretty obvious that nvidia does not care for linux users in this space, if you're paying and struggling to run their service while they are doing nothing to support you the customer, quite frankly they don't deserve your money or attention.
